src/main/java/core/basesyntax/figure/producers/FigureProducer.java

```diff
@@ -0,0 +1,56 @@
+package core.basesyntax.figure.producers;
+
+import core.basesyntax.figure.Figure;
+import core.basesyntax.figure.figures.Circle;
+import core.basesyntax.figure.figures.IsoscelesTrapezoid;
+import core.basesyntax.figure.figures.Rectangle;
+import core.basesyntax.figure.figures.RightTriangle;
+import core.basesyntax.figure.figures.Square;
+import java.util.Random;
+
+public class FigureProducer {
+
+    public Figure getRandomFigure() {
+        Random randomizer = new Random();
+        int bound = 1000;
+
+        switch (new Random().nextInt(5)) {
```

Comment:
if 5 is a number of your figures it's better to declare as a constant 
private static int FIGURE_NUMBER = 5;
this way it won't be a magic number in the code

Reply:
Ok, got it. Also had something like that on mind, I mean to get read of magic numbers.
Is there a way to find a number of classes who extends that class so the figure number will update it self?